### DISCUSSION TITLE
open ai请求发生错误

### DISCUSSION THREAD

**Discussion by Ethereal-sakura** [KroMiose/nonebot_plugin_naturel_gpt#114] [Q&A]
配置文件如图：
![image](https://user-images.githubusercontent.com/110333012/229289810-bca88527-e028-47e6-9cbf-36c5fbf8a571.png)
报错信息：
![image](https://user-images.githubusercontent.com/110333012/229289828-333b44ac-d15d-438e-a47f-b3ebcd6da064.png)

**Comment by KroMiose** [comment]
可能是代理问题，尝试填写代理服务器配置

**Reply by chenlaoda-big** [reply]
> 可能是代理问题，尝试填写代理服务器配
代理服务器咋搞啊，示例中的用不了
